x/claim/module.go

```diff
@@ -73,7 +73,9 @@ func (AppModuleBasic) RegisterRESTRoutes(clientCtx client.Context, rtr *mux.Rout
 
 // RegisterGRPCGatewayRoutes registers the gRPC Gateway routes for the module.
 func (AppModuleBasic) RegisterGRPCGatewayRoutes(clientCtx client.Context, mux *runtime.ServeMux) {
-	types.RegisterQueryHandlerClient(context.Background(), mux, types.NewQueryClient(clientCtx))
+	if err := types.RegisterQueryHandlerClient(context.Background(), mux, types.NewQueryClient(clientCtx)); err != nil {
```

Comment:
I'd prefer if we added a // ignore:errcheck, and got that working. Or at least leave `err := types.RegisterQuery...` on its own line, to maintain readability?

Reply:
https://golangci-lint.run/usage/false-positives/

We do // nolint:errcheck